buf.gen.yaml

```diff
@@ -0,0 +1,16 @@
+version: v1beta1
+
+plugins:
+  - name: go
+    out: .
+    ## The ",M<path/to/proto>=<package name>" below is required to tell
+    ## protoc-gen-go what the import path for a given proto file should be. This is
+    ## necessary when a proto file foo/a.proto imports a proto file from another
+    ## directory, e.g. bar/b.proto
+    opt: "plugins=grpc\
```

Comment:
We'll have a more generic solution to this problem soon :-)

Reply:
That's great! I only happened to know about this `M` flag business because of `prototool` adding it when generating protos for another project.